### PR TITLE
fix: share logs to Wire users (AR-3213)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3213" title="AR-3213" target="_blank">BugAR-3213</a>  PlayTest 14.03 - Can't share logs from Wire to Wire on Android 13
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

It's not possible to share logs internally in Wire app to other users.

### Causes (Optional)

We were not checking if have multiple share or not

### Solutions

_Briefly describe the solutions you have implemented for the issues explained above._

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1682/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.1(6277403)(6277403)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.1(6277403)(6277403)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaViewModel.kt [hunk 1]
```diff
@@ -233,18 +233,14 @@ class ImportMediaViewModel @Inject constructor(
...
+            // if stream count is 0 the type will be text, we check the type to double check if it is text
+            // todo : handle the text , we can get the text from incomingIntent.text
+        } else {
+            if (incomingIntent.isSingleShare) {
                 // ACTION_SEND
                 handleSingleIntent(incomingIntent, activity)
-            }
-
-            else -> {
+            } else {
```
Comment (borichellow) [inline app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaViewModel.kt:243]: can you move all the `if`s to the same level
```
if (incomingIntent.streamCount == 0) {

} else if (incomingIntent.isSingleShare) {

} else {

}
```

**Review by borichellow — APPROVED** [review]
nice catch, just a small code-style suggestion :)

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1682/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6284422)(6284422)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6284422)(6284422)-staging-compat.apk)

**Review by alexandreferris — APPROVED** [review]

**Pull request merged**